spec/sec-properties-of-the-regexp-prototype-object-patch.html

```diff
@@ -90,7 +90,7 @@ <h1>Runtime Semantics: RegExpBuiltinExec ( _R_, _S_ )</h1>
           1. <ins>Else,</ins>
             1. <ins>If _groupNames_ is a List, append *undefined* to _groupNames_.</ins>
         1. <ins>Let _indicesArray_ be MakeIndicesArray(_S_, _indices_, _groupNames_).</ins>
-        1. <ins>Perform ! DefinePropertyOrThrow(_A_, `"indices"`, PropertyDescriptor { [[Value]]: _indicesArray_, [[Writable]]: *false*, [[Enumerable]]: *false*, [[Configurable]]: *true* }).</ins>
+        1. <ins>Perform ! CreateDataProperty(_A_, `"indices"`, _indicesArray_).</ins>
```

Comment:
fwiw i filed https://github.com/tc39/ecma262/pull/1676 to convert all these to CreateDataPropertyOrThrow, since they shouldn't be able to fail.

Reply:
Do we need to fix it here or let it wait for https://github.com/tc39/ecma262/pull/1676?

Reply:
Fine to wait; once that merges, this one (and the spec pr) would need updating.